docs/examples/lmu.ipynb

```diff
@@ -0,0 +1,295 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with nengo.Network(seed=seed) as net:\n",
+    "    # remove some unnecessary features to speed up the training\n",
+    "    nengo_dl.configure_settings(\n",
+    "        trainable=None, stateful=False, keep_history=False,\n",
+    "    )\n",
+    "    \n",
+    "    # input node\n",
+    "    inp = nengo.Node(np.zeros(train_images.shape[-1]))\n",
+    "\n",
+    "    # lmu cell\n",
+    "    lmu = LMUCell(\n",
+    "        units=212, \n",
+    "        order=256, \n",
+    "        theta=train_images.shape[1], \n",
+    "        input_d=train_images.shape[-1]\n",
+    "    )\n",
+    "    conn = nengo.Connection(inp, lmu.x, synapse=None)\n",
+    "    net.config[conn].trainable = False\n",
+    "\n",
+    "    # dense linear readout\n",
+    "    out = nengo.Node(size_in=10)\n",
+    "    nengo.Connection(lmu.h, out, transform=nengo_dl.dists.Glorot(), synapse=None)\n",
+    "\n",
+    "    # record output. note that we set keep_history=False above, so this will\n",
+    "    # only record the output on the last timestep (which is all we need\n",
+    "    # on this task)\n",
+    "    p = nengo.Probe(out)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And now we can train the model. To save time in this example we will download some pretrained weights, but you can set `do_training=True` below to run the training yourself."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "do_training = False\n",
+    "\n",
+    "with nengo_dl.Simulator(\n",
+    "        net, minibatch_size=100, unroll_simulation=16) as sim:\n",
+    "    sim.compile(\n",
+    "        loss=tf.losses.SparseCategoricalCrossentropy(from_logits=True),\n",
+    "        optimizer=tf.optimizers.Adam(),\n",
+    "        metrics=[\"accuracy\"],\n",
+    "    )\n",
+    "\n",
+    "    print(\n",
+    "        \"Initial test accuracy: %.2f%%\" \n",
+    "        % (sim.evaluate(test_images, test_labels, verbose=0)[\"probe_accuracy\"] * 100)\n",
+    "    )\n",
+    "\n",
+    "    if do_training:\n",
+    "        sim.fit(train_images, train_labels, epochs=5)\n",
+    "        sim.save_params(\"./lmu_params\")\n",
+    "    else:\n",
+    "        urlretrieve(\n",
+    "            \"https://drive.google.com/uc?export=download&\"\n",
+    "            \"id=1qhKmpnipk8AK2bsilPlJBFQg1t7XAEZr\",\n",
+    "            \"lmu_params.npz\")\n",
+    "        sim.load_params(\"./lmu_params\")\n",
+    "\n",
+    "    print(\n",
+    "        \"Final test accuracy: %.2f%%\" \n",
+    "        % (sim.evaluate(test_images, test_labels, verbose=0)[\"probe_accuracy\"] * 100)\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can see that the network is achieving ~96% accuracy, which is state of the art performance on psMNIST."
+   ]
+  }
+ ],
+ "metadata": {
```

Comment:
I think a lot of this metadata should be stripped from the notebook (unless you leave it in for NengoDL).

Reply:
I haven't been bothering to strip it so far in any of the examples (but not opposed to it).

Reply:
If they're not stripped in other notebooks then not worth bothering with for now :+1: